docs/source/rbac/roles.md

```diff
@@ -0,0 +1,60 @@
+# Roles
+JupyterHub provides three **default roles** which are automatically loaded to the database at the startup:
+
+```{admonition} **Default roles**
+- **_user_** role carries single `all` scope that grants _least user access_ to perform only default user actions.
+- **_admin_** role contains all available scopes and grants full rights to all actions similarly to the current admin status.
+- **_server_** role allows for posting activity only through the single `users:activity` scope.
+```
+
+Roles can also be customly defined and assigned to users, services, groups and tokens. 
+
+**_Users_** and **_services_** are assigned a default role if no custom role is requested based on their admin status.
```

Comment:
```suggestion
**_Users_** and **_services_** are assigned a default role ( **_user_** or **_admin_**)  if no custom role is requested based on their admin status.
```
This suggestion is based on my understanding, if it's wrong then obviously reword as you think best!